examples/java/integration-test/rest/APIIntegrationTest.java

```diff
@@ -7,10 +7,10 @@
 import com.twilio.rest.api.v2010.Account;
 import com.twilio.rest.api.v2010.AccountReader;
 import com.twilio.rest.api.v2010.AccountUpdater;
-import com.twilio.rest.api.v2010.account.Call;
-import com.twilio.rest.api.v2010.account.CallCreator;
-import com.twilio.rest.api.v2010.account.CallDeleter;
-import com.twilio.rest.api.v2010.account.CallFetcher;
+import com.twilio.rest.api.v2010.Call;
+import com.twilio.rest.api.v2010.CallCreator;
+import com.twilio.rest.api.v2010.CallDeleter;
+import com.twilio.rest.api.v2010.CallFetcher;
```

Comment:
Shouldn't this be under `account`?

https://github.com/twilio/twilio-java/blob/main/src/main/java/com/twilio/rest/api/v2010/account/CallCreator.java